# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/CacheIR.h
# Commit: 6042c5c7c2a0
# Full Hash: 6042c5c7c2a02d1fdc08354b801ed6b89da67bd3
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2020-05-07 21:43:47
# Regressor Bug: 1635878
# File Overlap Count: 1
# Description:
#   Bug 1635878 - CacheIR: Optimize TypedArray length. r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74138
# ==============================================================================

diff -r 762b51f271f4 -r 6042c5c7c2a0 js/src/jit/CacheIR.h
--- a/js/src/jit/CacheIR.h	Thu May 07 18:02:49 2020 +0000
+++ b/js/src/jit/CacheIR.h	Thu May 07 12:21:38 2020 +0000
@@ -1081,6 +1081,8 @@
                                       HandleId id);
   AttachDecision tryAttachObjectLength(HandleObject obj, ObjOperandId objId,
                                        HandleId id);
+  AttachDecision tryAttachTypedArrayLength(HandleObject obj, ObjOperandId objId,
+                                           HandleId id);
   AttachDecision tryAttachModuleNamespace(HandleObject obj, ObjOperandId objId,
                                           HandleId id);
   AttachDecision tryAttachWindowProxy(HandleObject obj, ObjOperandId objId,